pkgs/tools/admin/ansible/1.9.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchurl
+, fetchFromGitHub
+, pythonPackages
+, windowsSupport ? false
+}:
+
+with pythonPackages;
+
+let
+  jinja = jinja2.overridePythonAttrs (old: rec {
+    version = "2.8.1";
+    name = "${old.pname}-${version}";
+    src = fetchFromGitHub {
+      owner = "pallets";
+      repo = "jinja";
+      rev = version;
+      sha256 = "0m6g6fx6flxb6hrkw757mbx1gxyrmj50w27m2afdsvmvz0zpdi2a";
+    };
+  });
+in buildPythonPackage rec {
+  version = "1.9.4";
```

Comment:
Sorry this is an unsupported version of ansible: https://docs.ansible.com/ansible/2.4/release_and_maintenance.html#release-status

If you still need this version, you can put it into package overrides or a `default.nix` for consumption in nix-shell.

Reply:
Ok